code-challenges.js

```diff
@@ -1,34 +1,112 @@
-// ASSESSMENT 3: Coding Practical Questions
-
-
-// --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
-
-
-
-
-// --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
-
-var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
-// Expected output: [-9, 9, 7, 199]
-var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
-// Expected output: [-823, 7, 23]
-
-
+// // ASSESSMENT 3: Coding Practical Questions
+
+
+// // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
+// //research what on earth Fibonacci sequence is
+
+// //draft up mock test w/Jest 
+// // describe("fibo", () => {
+// //     test("return the first 10 numbers of the Fibonacci sequence in an array", () => {
+// //         expect(fibo(n)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
```

Comment:
you'd want this test input to be specific to instead of `n` you'd want to put an actual number. but otherwise woo hoo!